src/kakarot/accounts/contract/library.cairo

```diff
@@ -23,7 +23,7 @@ func bytecode_len_() -> (res: felt) {
 }
 
 @storage_var
-func storage_(key: Uint256) -> (value: Uint256) {
+func storage_(key: felt) -> (value: Uint256) {
```

Comment:
since here the storage variable is named storage_

I think it'll never be used by our utils, since it's location in storage is
pedersen(sn_keccak("storage_"), felt)

Reply:
can you explain more?